.travis.yml

```diff
@@ -61,10 +61,21 @@ matrix:
           packages:
             - gcc-multilib
             - libgmp-dev:i386
+    - compiler: gcc
+      env:
+        - FIELD=auto  BIGNUM=no  SCALAR=auto  ENDOMORPHISM=yes  STATICPRECOMPUTATION=yes  ASM=x86_64
```

Comment:
Imo easier to read if you don't redefined variables like field and scalar again.